common-arcana.lic

```diff
@@ -241,6 +241,48 @@ module DRCA
     end
   end
 
+  def update_astral_data(data)
+    if data['moon']
+      data = set_moon_data(data)
+    elsif data['stats']
+      data = set_planet_data(data)
+    end
+    data
+  end
+
+  def find_visible_planets(planets)
+    return if DRC.bput('get my telescope', 'You get', 'What were you', 'You are already') == 'What were you'
```

Comment:
getting and stowing a telescope is done in astrology.lic as well. I could see the argument for a common-guildname based on this

Reply:
Seems like a future PR